src/compas_ui/rhino/forms/scene.py

```diff
@@ -50,6 +51,39 @@ def on_click(sender, e):
         return control
 
 
+class PushCell(Eto.Forms.CustomCell):
```

Comment:
Parts about forms are outdated, will get this updated once we finalize the `Project` class